src/Nelmio/Alice/Instances/Processor/Methods/ArrayValue.php

```diff
@@ -44,6 +44,7 @@ public function canProcess(ProcessableInterface $processable)
      */
     public function process(ProcessableInterface $processable, array $variables)
     {
+        /** @var array $values */
         $values = $processable->getValue();
```

Comment:
Blame @tshelburne it is his code )) I just added docblock to clearify that in this case value will be an array.